lib/api.js

```diff
@@ -57,12 +57,32 @@ function addType(chrome, domainName, type) {
     chrome[domainName][type.id] = help;
 }
 
+function addDomainListener(chrome, domainName) {
+    const handler = (_eventName, handler) => {
```

Comment:
Not sure about the name here. It is an eventName but the full name is also an eventName.
I could do this:
```
const handler = (eventName, handler) => {
  eventName = `${domainName}.${eventName}`; 
```

But then again reassigning arguments is a bad practice if I'm not mistaken